about.html

```diff
@@ -0,0 +1,90 @@
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="">Program</a></li>
+                        <li><a href="">Join</a></li>
+                        <li><a href="">Sponsor</a></li>
+                        <li><a href="">News</a></li>
+                        <li><a href="">Campiagn</a></li>
+                    </ul>   
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="about-main-section primary-font">
+            <div class="about-info container">
+                <button class="nav-btn"><i class="fa-solid fa-bars fa-2x"></i></button>
+                <p class="sub-title">"Kabul Mesuem Exhibition"</p>
+                <h1 class="title">FOURTH ROUND OF KABUL MESUEN EXHABITION</h1>
+                <hr class="title-underline">
+                <div class="about-short-info">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer congue mollis malesuada. Mauris enim massa, interdum dictum vestibulum at, pretium vitae libero. Morbi libero tortor, luctus tincidunt purus et, elementum tempus nibh. Etiam elit nisl, sagittis quis consequat sit amet, congue vitae turpis. In pellentesque aliquam felis eu varius. Pellentesque interdum nisl ut justo tempor tincidunt. Ut sodales vel quam a semper.</p>
+                    <p>Duis accumsan venenatis eros. Nullam interdum neque lectus, eget pretium ligula vulputate nec. Proin et ipsum pharetra, lobortis est ut, suscipit nisi. Mauris blandit laoreet nibh. Mauris semper, orci ac iaculis egestas, magna dolor suscipit justo, varius vehicula purus quam quis enim. In consectetur pulvinar dui, ut vestibulum orci rhoncus et. Aliquam nec lacinia lacus.</p>
+                </div>
+                <p>Please contact us per Email for any further questions about Exhibition</p>
+                <a href="">kabulmesuem@afg.org</a>
+            </div>
+            <div class="container">
+                <h2>Kabul Mesuem Exhibition Event 2023</h2>
+                <hr class="title-underline">
+                <p class="exhibition-info">The logo of kabul Mesuem Exhibition 2023 was decided through the logo competition from 8.June to 7.July.</p>
+                <div class="exhibition-logo">
+                    <img class="afg-logo" src="image/afg-logo.png" alt="">
+                </div>
+            </div>
+            <hr class="container-sep">
+            <div class="past-event container">
+                <h2>See the past Exhibitions</h2>
+                <p>Take a look the last two Exhabition which took place in Darulaman Kabul.</p>
+                <div class="event">
+                    <h3>2020</h3>
+                    <p>Kabul Mesuem Exhibition 2020 in Darulman</p>
+                </div>
+                <div class="event">
+                    <h3>2022</h3>
+                    <p>Kabul Mesuem Exhibition 2020 in Darulman</p>
+                </div>
+            </div>
```

Comment:
- [ ] As the previous reviewer said, This past exhibition section is missing a couple of things.
- There is a missing red divider after the section title `See the past Exhibitions`. Please include it and ensure you give appropriate spacing.
- The images should be horizontally aligned on the desktop view and vertically aligned on mobile. Kindly implement it for desktop view. The images should be actual images with a red transparent layer on them.

> You stick the colors, align-items 

## Original design 
![233768369-e4bfcf88-741e-4581-9c32-82c083533bb8](https://user-images.githubusercontent.com/57408419/233783413-b8a856e9-dd3b-42ff-b4d3-c216aa04be40.png)

## Your design

![233768389-61e49ae4-6198-4a45-afdd-1fdf435353d8](https://user-images.githubusercontent.com/57408419/233783426-27de89c3-4e66-46da-a892-b4a79414989b.png)